tests/test_somersault.py

```diff
@@ -259,7 +259,7 @@ def test_code_table_params(self) -> None:
         self.assertEqual(
             decoded_resp_data["last_pos_response"][1]  # type: ignore[index, call-overload]
             ["num_flips_done"],  # type: ignore[index, call-overload]
-            bytes([123]))
+            123)
```

Comment:
Why?

Reply:
mainy to better allow non-byte aligned MatchingRequestParameters, but I think using integers here feels more natural, too... (and since python has arbitrary-sized integers, this does not reduce generality.)